### ISSUE TITLE
Custom Recipes do not work with CommandAPI

### ISSUE THREAD

**Issue by KieranGateley** [JorelAli/1.13-Command-API#118]
**CommandAPI version**
v3.3

**Minecraft version**
1.16.1

**Describe the bug**
Custom Recipes are being loaded in, then removed by commandapi on server load

**My code**
Command Registration can be found at https://github.com/CelesteMagisteel/ModdedAdditions/tree/master/src/main/java/xyz/fluxinc/moddedadditions/commands

Recipe Registration in https://github.com/CelesteMagisteel/ModdedAdditions/blob/master/src/main/java/xyz/fluxinc/moddedadditions/utils/CustomRecipeUtils.java

**Behavior**
The recipes should be loaded into the server and accessible to players. However, the code to register them runs, but the recipes no longer load. Removing the commands added by CommandAPI and the plugin itself causes the issue to be resolved.

**Comment by JorelAli** [comment]
This may sound like a complete pain, but does this issue arise on Minecraft 1.15.2?

**Comment by KieranGateley** [comment]
Seemingly works as intended on 1.15.2

**Comment by JorelAli** [comment]
> Seemingly works as intended on 1.15.2

@KieranGateley Thanks for taking the time to confirm this - it appears my predictions were correct. This appears to correlate to my fix for #114 due to force-reloading datapacks on 1.16+. While this is indeed an annoying setback, I'll try my best to see if I can get this resolved within the next week.

**Comment by KieranGateley** [comment]
Tahnk you

**Comment by JorelAli** [comment]
If you're desperate, a **temporary** one that _should_ work is to do the following:

```java
@Override
public void onEnable() {
    Bukkit.getScheduler().scheduleSyncDelayedTask(plugin, () -> {
        //register recipes here
    }, 20L);
}
```

**Comment by JorelAli** [comment]
I've confirmed that recipes work in the 3.4 release on Minecraft 1.16.1 by re-registering the recipes after the CommandAPI reloads datapacks. This does in no way affect other plugins, so it's all good!

Marking as solved since it's fixed for version 3.4 due to release within the next 24 hours.
